src/blocks/plugins/css-handler/index.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+import { debounce } from 'lodash';
+
+import apiFetch from '@wordpress/api-fetch';
+
+import {
+	dispatch,
+	select,
+	subscribe
+} from '@wordpress/data';
+
+const { createNotice } = dispatch( 'core/notices' );
+
+const savePostMeta = debounce( async() => {
+	const { getCurrentPostId } = select( 'core/editor' );
+	const postId = getCurrentPostId();
+
+	createNotice(
+		'info',
+		__( 'Saving CSS…', 'otter-blocks' ),
+		{
+			isDismissible: true,
+			type: 'snackbar'
+		}
+	);
+
+	await apiFetch({ path: `themeisle-gutenberg-blocks/v1/save_post_meta/${ postId }`, method: 'POST' });
+
+	createNotice(
+		'info',
+		__( 'CSS saved.', 'otter-blocks' ),
+		{
+			isDismissible: true,
+			type: 'snackbar'
+		}
+	);
+}, 5000 );
+
+const saveWidgets = debounce( async() => {
+	createNotice(
+		'info',
+		__( 'Saving CSS…', 'otter-blocks' ),
+		{
+			isDismissible: true,
+			type: 'snackbar'
+		}
+	);
+
+	await apiFetch({ path: 'themeisle-gutenberg-blocks/v1/save_widgets_styles', method: 'POST' });
+
+	createNotice(
+		'info',
+		__( 'CSS saved.', 'otter-blocks' ),
+		{
+			isDismissible: true,
+			type: 'snackbar'
+		}
+	);
+}, 5000 );
+
+let reusableBlocks = {};
+
+subscribe( () => {
+	if ( select( 'core/edit-widgets' ) ) {
+		const {
+			isSavingWidgetAreas,
+			getEditedWidgetAreas
+		} = select( 'core/edit-widgets' );
+
+		const isSavingWidgets = isSavingWidgetAreas();
+		const editedAreas = getEditedWidgetAreas();
+
+		if ( isSavingWidgets && 0 < editedAreas.length ) {
+			saveWidgets();
+		}
+	}
+
+	if ( select( 'core/editor' ) ) {
+		const {
+			isCurrentPostPublished,
+			getEditedPostAttribute,
+			isSavingPost,
+			isPublishingPost,
+			isAutosavingPost,
+			__experimentalIsSavingReusableBlock
+		} = select( 'core/editor' );
+
+		const { __experimentalReusableBlocks } = select( 'core/block-editor' ).getSettings();
+
+		const { isSavingEntityRecord } = select( 'core' );
+
+		const { getBlocks } = select( 'core/block-editor' );
+
+		const { editPost } = dispatch( 'core/editor' );
+
+		const meta = getEditedPostAttribute( 'meta' ) || {};
+
+		if ( undefined !== meta._themeisle_gutenberg_block_has_review ) {
```

Comment:
The property of this meta tag is undefined, thus the review block will not be marked to exists in the post.

Reply:
I took a look at the DB post meta and it seams that our meta value is not registered.

![image](https://user-images.githubusercontent.com/17597852/137283695-d89f2eed-552f-4e17-bb1b-068d78ed2cc2.png)